CoreWebServer/src/au/gov/asd/tac/constellation/webserver/impl/GraphImpl.java

```diff
@@ -71,8 +71,9 @@
      *
      * @throws IOException
      */
-    public static void get_get(final OutputStream out) throws IOException {
-        final Graph graph = GraphManager.getDefault().getActiveGraph();
+    public static void get_get(final String graphId, final OutputStream out) throws IOException {
+        final Graph graph = graphId==null ? RestUtilities.getActiveGraph() : GraphNode.getGraph(graphId);
+        System.out.printf("GRAPHID %s %s\n", graphId, graph);
```

Comment:
I'll assume this was added during debugging so can you remove this line?
If you wanted to log this then please use `java.util.logger` as we are trying to standardise on that. `System.out.printf` won't appear in the log file.

Reply:
I always forget the printfs.